hokusai/production.yml

```diff
@@ -33,170 +33,22 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 5000
-          # readinessProbe:
-          #   httpGet:
-          #     port: 5000
-          #     path: /system/up
-          #     httpHeaders:
-          #       - name: X-FORWARDED-PROTO
-          #         value: https
-          #   initialDelaySeconds: 5
-          #   periodSeconds: 5
+          readinessProbe:
+            httpGet:
+              port: 5000
+              path: /system/up
+              httpHeaders:
+                - name: X-FORWARDED-PROTO
+                  value: https
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            timeoutSeconds: 10
```

Comment:
This should help with Pods occasionally falling over when Kubernete's can't make the health check - i.e. `Client connection closed while awaiting headers` kinda errors